setup.py

```diff
@@ -1079,6 +1079,7 @@ def print_box(msg):
                 'utils/model_dump/code.js',
                 'utils/model_dump/*.mjs',
             ],
+            'torchgen': [],
```

Comment:
Do we have tests that do "import torchgen" already?

Reply:
Ah good point. Yes, your land-then-check plan sounds reasonable -- we can punt something like a binary dist test to the future (or just rely on e.g. functorch importing torchgen as the test)